### ISSUE TITLE
tmuxinator and bundler fail

### ISSUE THREAD

**Issue by aklt** [NixOS/nixpkgs#16648]
## Issue description

I have installed tmuxinator by adding it to /etc/nixos/configuration.nix
I have also added bundler to configuration.nix

Attempting to run it gives me:

    # tmuxinator 
    /nix/store/n5gqlhs1nmadxkbibv42wnxq3m7f404k-ruby-2.3.0-p0/lib/ruby/2.3.0/rubygems.rb:241:in `bin_path': can't find gem bundler (>= 0) (Gem::GemNotFoundException)
        from /run/current-system/sw/bin/bundle:14:in `<main>'

### Steps to reproduce

As mentioned above 

## Technical details

* System: (NixOS: `16.03.973.8bf31d7 (Emu) `)
* Nix version: (`nix-env (Nix) 1.11.2`)
* Nixpkgs version: (`16.03.973.8bf31d7`)

**Comment by ericsagnes** [comment]
This should have been fixed with the last update of tmuxinator (#15870) on the unstable channel.

Could you try it with the following command and check if it works?

```
$ nix-shell -p tmuxinator -I nixpkgs=https://github.com/NixOS/nixpkgs-channels/archive/nixos-unstable.tar.gz --pure
```

**Comment by aklt** [comment]
I just tested this on a new account on my machine and tmuxinator seems to be running fine.
Thanks.
